projects/storefrontstyles/scss/components/organization/budgets/_budgets-list.scss

```diff
@@ -0,0 +1,25 @@
+%cx-budgets-list {
+  .cx-textalign {
+    @include media-breakpoint-down(xs) {
+      text-align: center;
+      margin: auto;
+    }
+  }
+
+  .table {

```

Comment:
If we want to re-use cx-table and avoid of code duplication, this part of styles should be for every cx-table. It is new component so need to be added